guides/source/6_0_release_notes.md

```diff
@@ -211,10 +211,240 @@ Please refer to the [Changelog][active-record] for detailed changes.
+
+*   Remove deprecated `ActiveRecord::ConnectionAdapters::SQLite3Adapter#valid_alter_table_type?`.
+    ([Commit](https://github.com/rails/rails/commit/45b4d5f81f0c0ca72c18d0dea4a3a7b2ecc589bf))
+
+*   Remove support for passing the column name to `sum` when a block is passed.
+    ([Commit](https://github.com/rails/rails/commit/91ddb30083430622188d76eb9f29b78131df67f9))
+
+*   Remove support for passing the column name to `count` when a block is passed.
+    ([Commit](https://github.com/rails/rails/commit/67356f2034ab41305af7218f7c8b2fee2d614129))
+
+*   Remove support for delegation of missing methods in a relation to arel.
+    ([Commit](https://github.com/rails/rails/commit/d97980a16d76ad190042b4d8578109714e9c53d0))
+
+*   Remove support for delegating missing methods in a relation to private methods of the class.
+    ([Commit](https://github.com/rails/rails/commit/a7becf147afc85c354e5cfa519911a948d25fc4d))
+
+*   Remove support for specifying a timestamp name for `#cache_key`.
+    ([Commit](https://github.com/rails/rails/commit/0bef23e630f62e38f20b5ae1d1d5dbfb087050ea))
+
+*   Remove deprecated `ActiveRecord::Migrator.migrations_path=`.
+    ([Commit](https://github.com/rails/rails/commit/90d7842186591cae364fab3320b524e4d31a7d7d))
+
+*   Remove deprecated `expand_hash_conditions_for_aggregates`.
+    ([Commit](https://github.com/rails/rails/commit/27b252d6a85e300c7236d034d55ec8e44f57a83e))
+
+
 ### Deprecations
 
+*   Deprecate mismatched case-sensitivity collation comparisons for uniqueness validator.
+    ([Commit](https://github.com/rails/rails/commit/9def05385f1cfa41924bb93daa187615e88c95b9))
+
+*   Deprecate using class level querying methods if the receiver scope has leaked.
+    ([Commit](https://github.com/rails/rails/commit/4c6171d6056a346a953792a954853d14c000dc90))
+
+*   Deprecate `config.activerecord.sqlite3.represent_boolean_as_integer`.
+    ([Commit](https://github.com/rails/rails/commit/f59b08119bc0c01a00561d38279b124abc82561b))
+
+*   Deprecate passing `migrations_paths` to `connection.assume_migrated_upto_version`.
+    ([Commit](https://github.com/rails/rails/commit/c1b14aded27e063ead32fa911aa53163d7cfc21a))
+
+*   Deprecate `ActiveRecord::Result#to_hash` in favor of `ActiveRecord::Result#to_a`.
+    ([Commit](https://github.com/rails/rails/commit/16510d609c601aa7d466809f3073ec3313e08937))
+
+*   Deprecate methods in `DatabaseLimits`: `column_name_length`, `table_name_length`,
+    `columns_per_table`, `indexes_per_table`, `columns_per_multicolumn_index`,
+    `sql_query_length`, and `joins_per_query`.
+    ([Commit](https://github.com/rails/rails/commit/e0a1235f7df0fa193c7e299a5adee88db246b44f))
+
+*   Deprecate `update_attributes`/`!` in favor of `update`/`!`.
+    ([Commit](https://github.com/rails/rails/commit/5645149d3a27054450bd1130ff5715504638a5f5))
+
 ### Notable changes
 
+*   Add `rails db:prepare` to migrate or setup a database by creates the database, loading the schema,
```

Comment:
We don't need `by creates the database` right?

Reply:
I re-reviewed the PR and I've updated the description to:

> Add `rails db:prepare` to create a database if it doesn't exist, and run its migrations.

Does that sound right to you?